esi_ui/static/dashboard/esi/lessee/nodes/provisioning/provisioning.module.js

```diff
@@ -2,14 +2,15 @@
   'use strict';
 
   angular
-    .module('horizon.dashboard.esi.lessee.nodes.provisioning', [])
+    .module('horizon.dashboard.project.esi.nodes.provisioning', [])
     .config(config)
-    .constant('horizon.dashboard.esi.lessee.nodes.provisioning.modal-spec', {
+    .constant('horizon.dashboard.project.esi.nodes.provisioning.modal-spec', {
```

Comment:
I can't seem to get the ESI ui panels up with this PR. I know Austin did some reorganization; is it possible these line changes are incorrect? Looking at the code, the other static js files all use `horizon.dashboard.esi.lessee`

Reply:
okay let me double check. what changes happened after reorganization because everything is working on my end.

Reply:
In your local checkout, did you rebase against the main branch?

Reply:
Usually whenever the directory tree gets restructured, I'd have to rm -r horizon/static for a "fresh" set of static files